src/library/styles/components/_textarea.scss

```diff
@@ -0,0 +1,60 @@
+.bsds-textarea {
+  @extend %field-control;
+  max-width: 100%;
+  min-width: 100%;
+  min-height: space(12x);
+  border-color: $neutral-40;
+
+  &::placeholder {
+    color: $text-color-subtle;
+  }
+
+  &:hover,
+  &:active {
+    border-color: $border-color-action;
+  }
+
+  &:read-only {
+    background: $neutral-95;
+
+    &:hover {
+      border-color: $border-color-action;
+      cursor: not-allowed;
```

Comment:
this combo feels a little funky. looks like text input isn't hovering to a blue border. we may want to chat with catie if that's specd, but i think we should keep the grey border here and align with text input.